documentation/readme.txt

```diff
@@ -0,0 +1 @@
+This folder will contain the plug-in specific documentation.

```

Comment:
Please use file name with *.md instead of *.txt such that wiki markup can be used.